File: wifi4eu-common/src/main/java/wifi4eu/wifi4eu/common/dto/model/AccessPointDTO.java
package wifi4eu.wifi4eu.common.dto.model;

public class AccessPointDTO {
    private int accessPointId;
    private String name;
    private String serialNumber;
    private String productName;
    private String modelNumber;
    private int installationId;
    private boolean indoor;

    public AccessPointDTO() {
    }

    public AccessPointDTO(int accessPointId, String name, String serialNumber, String productName, String modelNumber, int installationId, boolean indoor) {
        this.accessPointId = accessPointId;
        this.name = name;
        this.serialNumber = serialNumber;
        this.productName = productName;
        this.modelNumber = modelNumber;
        this.installationId = installationId;
        this.indoor = indoor;
    }

    public int getAccessPointId() {
        return accessPointId;
    }

    public void setAccessPointId(int accessPointId) {
        this.accessPointId = accessPointId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getSerialNumber() {
        return serialNumber;
    }

    public void setSerialNumber(String serialNumber) {
        this.serialNumber = serialNumber;
    }

    public String getProductName() {
        return productName;
    }

    public void setProductName(String productName) {
        this.productName = productName;
    }

    public String getModelNumber() {
        return modelNumber;
    }

    public void setModelNumber(String modelNumber) {
        this.modelNumber = modelNumber;
    }

    public int getInstallationId() {
        return installationId;
    }

    public void setInstallationId(int installationId) {
        this.installationId = installationId;
    }

    public boolean isIndoor() {
        return indoor;
    }

    public void setIndoor(boolean indoor) {
        this.indoor = indoor;
    }
}

File: wifi4eu-abac/src/main/angular/src/app/shared/swagger/model/AccessPointDTO.ts
import * as models from './models';
import {Type} from "class-transformer";

export interface AccessPointDTO {
    accessPointId?: number;

    name?: string;

    serialNumber?: string;

    productName?: string;

    modelNumber?: string;

    installationId?: number;

    indoor?: boolean;

}




/*default implementation one might extend from (or use as is) */


export class AccessPointDTOBase  implements AccessPointDTO{

    accessPointId?:  number ;


    name?:  string ;


    serialNumber?:  string ;


    productName?:  string ;


    modelNumber?:  string ;


    installationId?:  number ;


    indoor?:  boolean ;

}
